types/countdown/index.d.ts

```diff
@@ -1,11 +1,11 @@
 // Type definitions for countdown.js
 // Project: http://countdownjs.org/
-// Definitions by: Gabriel Juchault <https://github.com/gjuchault>
+// Definitions by: Gabriel Juchault <https://github.com/gjuchault>, Ionaru <https://github.com/Ionaru>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare namespace countdown {
   type DateFunction = (timespan: Timespan) => void;
-  type DateTime = number | Date | DateFunction;
+  type DateTime = number | Date | DateFunction | null | undefined;
```

Comment:
It might be a little cleaner to match the type to the docs: `type DateTime = number | Date | DateFunction | null` and let the `undefined` flow in from the optionality of `start` and `end` in the call definition of `CountdownStatic`. This would require `start` becoming optional there as well.